src/oc_erchef/raml-docs/organizations/search.yaml

```diff
@@ -1,8 +1,108 @@
+description: |
+  A list of the indexes that can be searched. The indices include
```

Comment:
@marcparadise the endpoint declaration is in index.raml (that's just the way it has to be broken up, AFAICT).